waspc/examples/crud-testing/src/client/MainPage.tsx

```diff
@@ -1,13 +1,16 @@
 import "./Main.css";
 
 import React, { useState } from "react";
-import { Link } from "react-router-dom";
+import { Link, routes } from "@wasp/router";
 
 import { tasks as tasksCrud } from "@wasp/crud/tasks";
-import { User, Task } from "@wasp/entities";
+import { User } from "@wasp/entities";
 
 const MainPage = ({ user }: { user: User }) => {
   const { data: tasks, isLoading } = tasksCrud.getAll.useQuery();
+
+  type Task = NonNullable<typeof tasks>[0];
```

Comment:
Nice trick to avoid importing the entity.

Small tip: Writing `number` instead of `0` is more idiomatic, but doesn't really change anything.

Reply:
Nice stuff, I didn't know that 👍